runtime/android/core/src/org/xwalk/core/XWalkActivityDelegate.java

```diff
@@ -104,25 +151,33 @@ public void onActivateFailed() {
         mIsInitializing = false;
 
         if (mXWalkUpdater.updateXWalkRuntime()) {
-            // Set the background to screen_background_dark temporarily if the default background
-            // is null in order to avoid the visual artifacts around the alert dialog
-            Window window = mActivity.getWindow();
-            if (window != null && window.getDecorView().getBackground() == null) {
-                Log.d(TAG, "Set the background to screen_background_dark");
-                window.setBackgroundDrawableResource(android.R.drawable.screen_background_dark);
-                mBackgroundDecorated = true;
+            if (!mIsDownloadMode) {
```

Comment:
I think `if (mXWalkUpdater.updateXWalkRuntime() && !mIsDownloadMode)` is enough. It reduces changed code